CMakeLists.txt

```diff
@@ -164,7 +164,7 @@ if(KernelArchARM)
     endif()
 endif()
 if(KernelArchRiscV)
-    KernelCommonFlags(-mcmodel=medany)
+    KernelCommonFlags(-mcmodel=medany -msmall-data-limit=64 -fno-common)
```

Comment:
Doing some of my own experiments, I found that the small-data-limit could be increased to 1024 before the small section exceeded 0x1000 in size and the final instruction count ended up at 14640 down from 15062.  What are your thoughts about this?